### PR TITLE
RTEMIS-138: Theming updates in about us page and menu.

### PR DESCRIPTION
**Pull request by sandee-mahlawat-innoraft**
Ticket # : https://innoraft.atlassian.net/browse/RTEMIS-138

---
### What the PR does
`(Explain fixes, changes and impacts)`
- Fixes/changes/Impacts ...

---
### What I have tested
`(Explain usecases you have tested locally)`
- ex: State Admin: Configuation for USIDE code.
- ex: App Admin: Enables multilingual configuration.

---
### Extras
`(Add Screenshots or videos)`

### CODE REVIEW

File: modules/rte_mis_gin/js/script.js [hunk 1]
```diff
@@ -5,11 +5,48 @@
...
+                    var collapsedMenu = $(this).find('.collapsed-menu');
+                    if (collapsedMenu.length > 0) {
+                        console.log('Found .collapsed-menu element:', collapsedMenu);
+                        collapsedMenu.toggleClass('abcxyz');
+                        console.log('Toggled class .abcxyz');
+                    } else {
+                        console.log('No .collapsed-menu element found');
+                    }
+                    $(this).toggleClass('active');
+                    console.log('Toggled class .active on .menu-item--expanded');
```
Comment (kunal-sin) [inline modules/rte_mis_gin/js/script.js:29]: Please remove console logs.

File: modules/rte_mis_gin/js/script.js [hunk 2]
```diff
@@ -5,11 +5,48 @@
...
                 $(this).toggleClass('active');
-                $('.main-menu-wrapper .menu').toggleClass('collapsed-menu');
+                $('.main-menu--wrapper .menu').toggleClass('collapsed-menu');
+                $('body').toggleClass('mobil-menu-active');
+            });            
+            $('.menu-item--expanded').click(function() {
+                var viewportWidth = $(window).width();
+                if (viewportWidth <= 992) {
+                    console.log('Clicked: .menu-item--expanded');
+                    console.log('Viewport width:', viewportWidth);
```
Comment (kunal-sin) [inline modules/rte_mis_gin/js/script.js:19]: Please remove console logs.

Comment (sandee-mahlawat-innoraft) [reply modules/rte_mis_gin/js/script.js]: Removed please review

File: modules/rte_mis_gin/js/script.js (hunk 1 above)
Comment (sandee-mahlawat-innoraft) [reply modules/rte_mis_gin/js/script.js]: Removed please review

**Pull request merged**